content/en/docs/setup/install/multicluster/single-network/verify/index.md

```diff
@@ -0,0 +1,318 @@
+   command:
+
+    {{< text bash >}}
+    $ kubectl get pod --context=${CTX_1} -n sample -l app=sleep
+    sleep-754684654f-n6bzf           2/2     Running   0          5s
+    {{< /text >}}
+
+1. Wait until the status of the `sleep` service is `Running`.
+1. Confirm the status of the `sleep` service on `Cluster_2` with the following
+   command:
+
+    {{< text bash >}}
+    $ kubectl get pod --context=${CTX_2} -n sample -l app=sleep
+    sleep-754684654f-dzl9j           2/2     Running   0          5s
+    {{< /text >}}
+
+1. After the status of the `sleep` service is `Running`, you can verify that
+   cross-cluster load balancing is working as expected.
+
+## Verify cross-cluster load balancing
+
+To verify that cross-cluster load balancing works as expected, call the
+HelloWorld service several times using the sleep service. To ensure load
+balancing is working properly, call the HelloWorld service from all clusters in
+your deployment.
+
+Using the previously [set environment variables,](/docs/setup/install/multicluster/#env-var)
+verify cross-cluster load balancing with following steps:
+
+1. Send one request on port 5000 from the `sleep` service on `Cluster_1` to the
+   `HelloWorld` service with the following command:
+
+    {{< text bash >}}
+    $ kubectl exec --context=${CTX_1} -it -n sample -c sleep \
+        $(kubectl get pod --context=${CTX_1} -n sample -l \
+            app=sleep -o jsonpath='{.items[0].metadata.name}') -- curl \
+            helloworld.sample:5000/hello
+    {{< /text >}}
+
+1. To verify that the traffic reaches the endpoints of the `HelloWorld` service
+   on your clusters, run the previous command multiple times.
+
+1. If the endpoints on both clusters are responding, the output shows both
+   `v1` and `v2` in the replies.
+
+    {{< text console >}}
+    Hello version: v2, instance: helloworld-v2-758dd55874-6x4t8
+    Hello version: v1, instance: helloworld-v1-86f77cd7bd-cpxhv
+    ...
+    {{< /text >}}
+
+1. Send a request on port 5000 from the `sleep` service on `Cluster_2` to the `HelloWorld`
+   service with the following command:
+
+    {{< text bash >}}
+    $ kubectl exec --context=${CTX_2} -it -n sample -c sleep \
+        $(kubectl get pod --context=${CTX_2} -n sample -l \
+            app=sleep -o jsonpath='{.items[0].metadata.name}') -- curl \
+            helloworld.sample:5000/hello
+    {{< /text >}}
+
+1. To verify that the traffic reaches the endpoints of the `HelloWorld` service
+   on your clusters, run the previous command multiple times.
+
+1. If the endpoints on both clusters are responding, the output shows both
+   `v1` and `v2` in the replies.
+
+    {{< text console >}}
+    Hello version: v2, instance: helloworld-v2-758dd55874-6x4t8
+    Hello version: v1, instance: helloworld-v1-86f77cd7bd-cpxhv
+    ...
+    {{< /text >}}
+
+**Congratulations!**
+
+You successfully verified the deployment of your multicluster mesh.
+
+## Clean up
```

Comment:
Is this not a production installation guide?
Why end it with "And now delete Istio?"

Reply:
We need to tell folks how to undo their deployment and reach the original state.